rush-plugins/rush-buildxl-graph-plugin/src/DropBuildGraphPlugin.ts

```diff
@@ -35,51 +45,67 @@ const DROP_GRAPH_PARAMETER_LONG_NAME: '--drop-graph' = '--drop-graph';
+      // TODO: Introduce an API to allow plugins to register command line options for arbitrary, existing commands
+      // in a repo
+      const dropGraphParameter: CommandLineStringParameter | undefined = context.customParameters.get(
+        DROP_GRAPH_PARAMETER_LONG_NAME
+      ) as CommandLineStringParameter;
+      if (!dropGraphParameter) {
+        // TODO: Introduce an API to allow plugins to register command line options for arbitrary, existing commands
+        // in a repo
+        throw new Error(
+          `The ${DROP_GRAPH_PARAMETER_LONG_NAME} parameter needs to be defined in "${RushConstants.commandLineFilename}" ` +
+            `and associated with the action "${commandName}"`
+        );
+      } else if (dropGraphParameter.kind !== CommandLineParameterKind.String) {
+        throw new Error(`The ${DROP_GRAPH_PARAMETER_LONG_NAME} parameter must be a string parameter`);
+      }
 
-          const dropGraphPath: string | undefined = dropGraphParameter.value;
-          if (dropGraphPath) {
-            const { dropGraphAsync } = await import('./dropGraph');
-            const isValid: boolean = await dropGraphAsync({
-              operations,
-              context,
-              dropGraphPath,
-              rushConfiguration,
-              logger: session.getLogger(PLUGIN_NAME)
-            });
+      const dropGraphPath: string | undefined = dropGraphParameter?.value;
+      if (dropGraphPath) {
+        const { dropGraphAsync } = await import('./dropGraph');
+        const isValid: boolean = await dropGraphAsync({
+          operations,
+          context,
+          dropGraphPath,
+          rushConfiguration,
+          logger: session.getLogger(PLUGIN_NAME)
+        });
 
-            if (!isValid) {
-              throw new Error('Failed to validate the graph');
-            } else {
-              // If the --drop-graph flag is present, we want to exit the process after dropping the graph
-              return new Set();
-            }
-          } else {
-            return operations;
-          }
+        if (!isValid) {
+          throw new Error('Failed to validate the graph');
+        } else {
+          // If the --drop-graph flag is present, we want to exit the process after dropping the graph
+          return new Set();
         }
-      );
-    });
+      } else {
+        return operations;
+      }
+    }
+
+    for (const buildXLCommandName of this._buildXLCommandNames) {
+      session.hooks.runPhasedCommand.for(buildXLCommandName).tap(PLUGIN_NAME, (command: IPhasedCommand) => {
+        command.hooks.createOperations.tapPromise(
+          {
+            name: PLUGIN_NAME,
+            stage: Number.MAX_SAFE_INTEGER // Run this after other plugins have created all operations
+          },
+          async (operations: Set<Operation>, context: ICreateOperationsContext) =>
+            await handleCreateOperationsForCommandAsync(buildXLCommandName, operations, context)
```

Comment:
```suggestion
            await handleCreateOperationsForCommandAsync(command.actionName, operations, context)
```
The name is one of the few properties that is available on `command`.